flow.py

```diff
@@ -209,6 +209,16 @@ def from_of_flow_stats(cls, of_flow_stats, switch):
                    actions=non_none_actions,
                    stats=FlowStats.from_of_flow_stats(of_flow_stats))
 
+    def __eq__(self, other):
+        include_id = False
+        if not isinstance(other, self.__class__):
+            raise ValueError(f'Error comparing two flows, the {other}'
+                             'is not a FlowBase class')
```

Comment:
There's a wrong assumption here that `self.__class__` will always be `FlowBase`. It'll be whatever class inherited from FlowBase (eg.: `v0x01.Flow`). Either you change the comparison to check just for FlowBase (and allow for comparing different classes of Flow, like comparing a v0x01 Flow against a v0x04) or you change the message to make it explicit that you won't allow comparing different versions of Flow (which I think makes more sense and suggested below). 

```suggestion
        if not isinstance(other, self.__class__):
            raise ValueError(f'Error comparing flows: {other} is not '
                             f'an instance of {self.__class__}')
```